providers/implementations/keymgmt/mlkem_kmgmt.c

```diff
@@ -196,7 +194,6 @@ static int mlkem_match(const void *keydata1, const void *keydata2, int selection
 }
 
 /* TODO(ML-KEM) as and when encode/decode becomes needed/standardized */
-#ifdef UNDEF
```

Comment:
Suggest rephrasing the comment as "...complete/review as and when...."

Reply:
I'll remove the comment, as we already have the TODO comment below